### PR TITLE
Add publish,makeConnectable and refCount,autoconnect

### PR DESCRIPTION
**Pull request by MortyMerr**

### CODE REVIEW

**Comment by freak4pc** [comment]
Thanks! can you run ./generate.rb ? :)

File: Data/operators.csv [hunk 1]
```diff
@@ -24,7 +24,9 @@ map,"map, tryMap",
 merge,"merge, tryMerge",
 multicast,multicast,
 observeOn,receive(on:),
+publish,makeConnectable
```
Comment (freak4pc) [inline Data/operators.csv]: Can you also make . sure you add the last comma so the last column is empty? (It's optional but I prefer keeping it standard)

**Comment by freak4pc** [comment]
Also, you'll have to rebase, I've made some changes :) 
Please make sure the operators in the CSV file are in their right place, alphabetically. Thank you!

File: Data/operators.csv (hunk 1 above)
Comment (MortyMerr) [reply Data/operators.csv]: Sure, done

**Comment by freak4pc** [comment]
Still conflicting, maybe it would be easier to take a clean `master`, do it, and run the script. You don't have to sort it, I added that to the script :)

File: Data/operators.csv [hunk 2]
```diff
@@ -3,8 +3,8 @@ asObservable(),eraseToAnyPublisher(),
 asObserver(),eraseToAnySubject(),
 bind(to:),`assign(to:on:)`,Assign uses a KeyPath which is really nice and useful. RxSwift needs a Binder / ObserverType to bind to.
 buffer,buffer,
-catchError,catch,
-catchErrorJustReturn,replaceError(with:),
+catchError,catch,catchError terminates sequence with error. catch completes as usual
+catchErrorJustReturn,replaceError(with:),catchErrorJustReturn terminates sequence error. replaceError(with:) completes as usual
```
Comment (freak4pc) [inline Data/operators.csv]: what does completes as usual mean? Finishes with `finished` or continues as if there was no error ?

Comment (MortyMerr) [reply Data/operators.csv]: Sorry, it's random commit. Deleted it

**Comment by MortyMerr** [comment]
Rebased and added last comma

**Pull request merged**